app/src/main/java/org/schabi/newpipe/player/MainVideoPlayer.java

```diff
@@ -394,6 +400,11 @@ protected void setShuffleButton(final ImageButton shuffleButton, final boolean s
         shuffleButton.setImageAlpha(shuffleAlpha);
     }
 
+    protected void setMuteButton(final ImageButton muteButton, final boolean isMuted) {
+        muteButton.setColorFilter(ContextCompat.getColor(getApplicationContext(), isMuted ? R.color.white : R.color.gray));
```

Comment:
What if the theme is white theme? The active button color should be black in that case. I'd suggest adding two attributes in `attrs.xml` named something along the lines of `button_active` and `button_inactive` and giving them different color values in the `themes.xml` file, so that they are themed.